cypress/integration/e2e.js

```diff
@@ -7,8 +7,8 @@ describe('Detects frameworks', () => {
     cy.contains('"name":"Next.js"')
   })
 
-  it('Next.js site framework detected on load - vanilla JS site', () => {
-    cy.contains('Vanilla JS Site').click()
-    cy.contains('"name":"Next.js"')
-  })
+  // it('Next.js site framework detected on load - vanilla JS site', () => {
+  //   cy.contains('Vanilla JS Site').click()
+  //   cy.contains('"name":"Next.js"')
+  // })
```

Comment:
In order to use pure ES modules, we need the Webpack output bundle to use ES modules instead of UMD. 
The "Vanilla JS site" relies on loading `framework-info` with a global variable. We need to remove those tests since they are now irrelevant. However, I am considering doing this as a separate, follow-up PR.